internal/dispatch/graph/check_test.go

```diff
@@ -178,7 +178,7 @@ func TestMaxDepth(t *testing.T) {
 	require.NoError(err)
 	require.True(revision.GreaterThan(decimal.Zero))
 
-	dispatch := NewLocalOnlyDispatcher()
+	dispatch := NewLocalOnlyDispatcher(10)
```

Comment:
I wonder if we should have a `New` function that sets a default

Reply:
There is a default for all non-test call sites.